setup.py

```diff
@@ -56,6 +56,8 @@ def read(fname):
         "flake8==3.9.2",
         "black==21.5b1",
         "isort==5.8.0",
+        "mypy>=0.900",
+        "mypy-zope @ https://github.com/Shoobx/mypy-zope/archive/391ce347b6c01dbc89e0194b62e47deadf53fbc6.tar.gz",
```

Comment:
Mypy-zope now has a release supporting mypy 0.902, and mypy itself nolonger ships types for pyyaml and mock; we have to add that separately

```suggestion
        "mypy-zope>=0.3.1",
        "types-PyYAML",
        "types-mock",
```